bundles/org.openhab.binding.lgwebos/src/main/java/org/openhab/binding/lgwebos/internal/handler/LGWebOSHandler.java

```diff
@@ -256,6 +273,9 @@ public void storeKey(@Nullable String key) {
 
     @Override
     public void storeProperties(Map<String, String> properties) {
+        if (disposed) {
```

Comment:
does this ever happen?

Reply:
This is the code from the PR #7299 So please consider moving your comment in the right PR.

This can happen because the websocket thread is closed asynchronously and so can call the thing handler (ConfigProvider) while the thing handler is already disposed.

Reply:
Removed from this PR